src/NuGet.Services.EndToEnd/Support/PackageDirectory.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.IO;
+using System.Linq;
+
+namespace NuGet.Services.EndToEnd.Support
+{
+    public class PackageDirectory
+    {
+        private readonly string _directory;
+
+        public PackageDirectory(string directory)
```

Comment:
Perhaps rename to `path`?
As such, [line 31](https://github.com/NuGet/NuGet.Services.EndToEnd/pull/10/files#diff-743a38bd7e4fa0d7296ba3955c27784aR31) will also return `directory._path` instead of `directory._directory` :)